## Задача 1

_Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256._  


$\underline{X}\pm Z_\frac{\alpha}{2}\cdot\frac{\sigma}{\sqrt{n}}$

$Z_\frac{\alpha}{2}=1.96$

$80\pm 1.96\cdot\frac{16}{\sqrt{256}} => [78.04;81.96]$

т.о. доверительный интервал $[78.04;81.96]$

## Задача 2

_В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные:_
> 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
>
_Предполагая, что результаты измерений подчинены нормальному закону  вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95._  

$\underline{X}\pm t_\frac{\alpha}{2}\cdot\frac{\sigma}{\sqrt{n}}$

In [1]:
import numpy as np
from scipy import stats

In [2]:
x = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
M = x.mean()
D = x.var(ddof=1)
t = stats.t.ppf(0.975, 9)
M - t * np.sqrt(D / 10), M + t * np.sqrt(D / 10)

(6.267515851415713, 6.912484148584288)

т.о. доверительный интервал $[6.268; 6.912]$

## Задача 3

_Рост дочерей 175, 167, 154, 174, 178, 148, 160, 167, 169, 170  
Рост матерей  178, 165, 165, 173, 168, 155, 160, 164, 178, 175  
Используя эти данные построить 95% доверительный интервал для разности среднего роста родителей и детей._


$\Delta\pm t_\frac{\alpha}{2}\cdot S_\Delta$

$\Delta=\underline{X_1}-\underline{X_2}$

$D_{общ}=\frac{1}{2}(D_1 + D_2)$

$S_\Delta=\sqrt{\frac{D_{общ}}{n_1}+\frac{D_{общ}}{n_2}}$

In [4]:
a = np.array([175, 167, 154, 174, 178, 148, 160, 167, 169, 170])
b = np.array([178, 165, 165, 173, 168, 155, 160, 164, 178, 175])

delta = a.mean() - b.mean()
D = (a.var(ddof=1) + b.var(ddof=1)) / 2
Se = np.sqrt(D / len(a) + D / len(b))
ns = (len(a) - 1) * 2
t = stats.t.ppf(0.975, ns)
delta - t * Se, delta + t * Se


(-10.068418034506857, 6.268418034506846)

т.о. доверительный интервал $[-10.068 ; 6.268]$